pkg/ts/Runtime.ts

```diff
@@ -8,15 +8,15 @@
 import {Arc} from './core/Arc.js';
 import {Host} from './core/Host.js';
 import {Store} from './core/Store.js';
+import {ArcMeta, ParticleMeta, StoreMeta} from './core/Meta.js';
```

Comment:
Can we do 
`import * from './core/types.js'`;
and use that as a pattern? e.g.
`import * from './utils/types.js'`;
?

Reply:
We can, but explicit import makes looking up the source of an identifier a constant time task (where the star can lead to a back tracking search through multiple source files for [potentially re-exported] symbols).

If you're not already using a language server for typescript (which can suggest missing imports without a re-build) I highly recommend it, but even by hand I think this approach is less work.

Reply:
Ah sorry the above is about the `import *`. The file name pattern makes sense to me (though can make things slightly harder to find I think the system has value).

Changed.
Cheers

Reply:
All of that is optimizations in Typescript-land, which I largely want to abandon in favor of runtime gains.

Reply:
It is unclear to me that `import *` has any particular benefits other than being concise (which is not a stated goal of our code). How would this help the runtime?

Reply:
Concision is indeed a goal, in the absence of other factors.

Reply:
Hmm, I seem to have been mistaken: https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import

We'd have to do: `import * as types from ...` instead. I'm leaving it for now

Reply:
👍